samples/other/run-from-junit/src/test/java/org/eclipse/dataspaceconnector/junit/EndToEndTest.java

```diff
@@ -161,6 +162,9 @@ private void loadNegotiation(ContractNegotiationStore negotiationStore) {
 
     @Provides(IdentityService.class)
     private static class TestServiceExtension implements ServiceExtension {
+        @Inject
+        private AssetLoader loader;
```

Comment:
needed?

Reply:
unfortunately yes, because the way default provides work is that they only get instantiated "on demand", and if there is no "demand" (which is what `@Inject` is), it won't get instantiated, thus not being provided by the `EdcExtension` resulting in an exception when running the test.

Reply:
@paullatzelsperger I think this test class is somewhat obsolete since now `system-tests` exist, I would delete it completely